cmd/root.go

```diff
@@ -156,6 +158,23 @@ func runRoot(cmd *cobra.Command, args []string) {
 		site, err = configureSiteAndLoadpoints(conf)
 	}
 
+	// setup telemetry
```

Comment:
Mir ist der Ablauf hier nicht ganz klar. Ist an dieser Stelle sichergestellt, dass `configureDatabase` aufgerufen wurde damit die Settings Calls in Telemetry auch einen Effekt haben?

Reply:
Ja, das passiert vorher im `configureEnvironment`.